parquet-avro/src/main/java/org/apache/parquet/avro/AvroSchemaConverter.java

```diff
@@ -54,7 +54,7 @@
 
   public static final String ADD_LIST_ELEMENT_RECORDS =
       "parquet.avro.add-list-element-records";
-  private static final boolean ADD_LIST_ELEMENT_RECORDS_DEFAULT = true;
+  private static final boolean ADD_LIST_ELEMENT_RECORDS_DEFAULT = false;
```

Comment:
Oops! I was making sure the compatibility tests passed when this is flipped, but didn't mean to change the default.